modules/optics/arrow-optics/src/main/kotlin/arrow/optics/Fold.kt

```diff
@@ -18,7 +18,8 @@ import arrow.typeclasses.monoid
  * @param S the source of a [Fold]
  * @param A the target of a [Fold]
  */
-interface Fold<S, A> {
+@higherkind
```

Comment:
I'm surprised that this works. I expected for @higherkind to be limited to classes.

Reply:
I was surprised 2 :D